packages/xo-web/src/common/xo/index.js

```diff
@@ -766,8 +766,19 @@ export const restartHost = (host, force = false) =>
     body: _('restartHostModalMessage'),
   }).then(
     () =>
-      _call('host.restart', { id: resolveId(host), force }).catch(error => {
-        if (noHostsAvailable.is(error)) {
+      _call('host.restart', { id: resolveId(host), force }).catch(async error => {
```

Comment:
would it be possible to group the confirm  instead of having 2 modal confirmation ?

Reply:
We are only reacting to xo-server errors.